src/utils/api/easeyAuthApi.js

```diff
@@ -1,66 +1,70 @@
 import config from "../../config";
 import axios from "axios";
+import { checkoutAPI } from "../../additional-functions/checkout";
+import { getCheckedOutLocations } from "./monitoringPlansApi";
 
-export const secureAxios = async (options) => {
+export const secureAxios = (options) => {
   options.headers = {
     authorization: `Bearer ${
       JSON.parse(sessionStorage.getItem("cdx_user")).token
     }`,
   };
 
-  return await axios(options);
+  return axios(options);
 };
 
 export async function authenticate(data_payload) {
-  try {
-    return await axios({
-      method: "POST",
-      url: `${config.services.authApi.uri}/authentication/sign-in`,
-      data: data_payload,
+  return axios({
+    method: "POST",
+    url: `${config.services.authApi.uri}/authentication/sign-in`,
+    data: data_payload,
+  })
+    .then((data_response) => {
+      console.log(data_response);
+      const { data } = data_response;
+      sessionStorage.setItem("cdx_user", JSON.stringify(data));
+      window.location.reload();
     })
-      .then((data_response) => {
-        console.log(data_response);
-        const { data } = data_response;
-        sessionStorage.setItem("cdx_user", JSON.stringify(data));
-        window.location.reload();
-      })
-      .catch((e) => {
-        throw e;
-      });
-  } catch (e) {
-    return {
-      success: false,
-      message: e.response.data.message,
-      statusCode: e.response.data.statusCode,
-      error: e.response.data,
-    };
-  }
+    .catch((e) => {
+      throw e;
+    });
 }
 
 export async function logOut(event) {
```

Comment:
I don't think this needs to be async.  since it doesn't return anything, nor await anything (other than what is done out of a callback [then()] function, which is its own thing), we can just treat it as a normal function